tests/login.test.js

```diff
@@ -0,0 +1,97 @@
+/*global describe*/
+/*global test*/
+/*eslint-env node*/
+const app = require('../app.js');
+const request = require('supertest');
+var cookie;
+
+describe('POST /login/', function() {
+	test('POST /login/ respond with json', function(done) {
+		request(app)
```

Comment:
see, the same login code all over again. you can create a module test-helpers and put it there.